containers/init/ready/ready.go

```diff
@@ -57,14 +58,22 @@ const OutputFileEnv = "READY_OUTPUT_FILE"
 // write all metadata.
 const OutputMetadataEnv = "METADATA_OUTPUT_FILE"
 
-// DefaultMetadataOutputFile is the name of the default file where the executable should
-// write the metadata.
-const DefaultMetadataOutputFile = "/tmp/metadata.json"
+// OutputNodeInfoEnv is the optional name of the file where the executable should
+// write the node information.
+const OutputNodeInfoEnv = "NODE_INFO_OUTPUT_FILE"
 
 // DefaultOutputFile is the name of the default file where the executable should
 // write the comma-separated list of IP addresses.
 const DefaultOutputFile = "/tmp/loadtest_workers"
 
+// DefaultMetadataOutputFile is the name of the default file where the executable should
+// write the metadata.
+const DefaultMetadataOutputFile = "/tmp/metadata.json"
+
+// DefaultNodeInfoOutputFile is the name of the default file where the executable should
+// write the node infomation.
+const DefaultNodeInfoOutputFile = "/tmp/node_info"
```

Comment:
when you output the file is it automatically knows the format? I had it with .json, not sure if it is okay to not have it.

Reply:
Yeah, the format should be json. By adding .json, it would be more clear. Thanks!